src/service/viewer-impl.js

```diff
@@ -742,7 +742,7 @@ export class Viewer {
    * Triggers "documentLoaded" event for the viewer.
    */
   postDocumentReady() {
-    this.sendMessageUnreliable_('documentLoaded', {
+    this.sendMessageUnreliable('documentLoaded', {
```

Comment:
After discussion, de-dup logic is actually hard to implement outside this class.

I made change to `sendMessageUnreliable` so that it will have a consistent return value. And I renamed it to `sendMessageCancelUnsent`. Let me know if this makes sense.